docs/ContainerizedSetup.asciidoc

```diff
@@ -131,39 +137,34 @@ ports:
   - "80:9526"
 ```
 
-### Enable the SSL access to the load balancer (optional)
+==== Enable the SSL access to the load balancer (optional)
 
 Enable the SSL access in three steps:
 
-1. To expose the SSL port uncomment this line in the docker-compose.yaml file
-   in the service nginx.
-
-```
-- "443:443"
-```
-
-You can change the exported port if 443 is already used in your computer, for
-instance:
-
-```
-- "10443:443"
-```
-
-2. Provide an SSL certificate.
-
-```
-- cert.crt:/etc/ssl/certs/openqa.crt
-- cert.key:/etc/ssl/certs/openqa.key
-```
+1. To expose the SSL port, uncomment this line in the docker-compose.yaml file
+   in the service nginx:
+   ```
+   - "443:443"
+   ```
+   You can change the exported port if 443 is already used in your computer, for
+   instance:
+   ```
+   - "10443:443"
+   ```
+
+2. Provide an SSL certificate:
+   ```
```

Comment:
This volumes has to be in the section volumes of the docker-compose service but this section is currently missing on the yaml file and cannot be included if there are not volumes. So please include "volumes:" before these two lines.

Reply:
The YAML looks like this:

```
    ports:
      - "9526:9526"
      # Uncomment for SSL access
      # - "443:443"
    environment:
      replicas: ${OPENQA_WEBUI_REPLICAS}
      # If is needed to set a certificate for SSL
      # - cert.crt:/etc/ssl/certs/opeqa.crt
      # - cert.key:/etc/ssl/certs/opeqa.key
```

So I would assume the documentation's first point asks me to uncomment the first comment line and the second point asks me to uncomment the two last comments. Not sure how volumes would come into play here. How would the YAML look like after following step 2? Maybe we should change the YAML so one has to uncomment lines if that's not already possible.

Reply:
I add this change here https://github.com/os-autoinst/openQA/pull/3821#issuecomment-811774254 but I am not sure is the place, feel free to add this here

Reply:
I would keep my PR as-is and when you do changes which require documentation updates you should to these documentation updates as part of these changes.

Reply:
ok